apps/pages/templates/pages/review_guide.html

```diff
@@ -94,14 +83,12 @@ <h2 id="tips">{{ _('Some tips for writing a great review') }}</h2>
     <ul>
       <li>
         {% trans %}
-          Submit simple reviews such as "Great!", "wonderful", or "bad" with
-          no further explanation.
+          Submit one-word reviews such as "Great!", "wonderful", or "bad".
```

Comment:
Not always! http://www.quickanddirtytips.com/education/grammar/how-to-use-quotation-marks